inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/release.adoc

```diff
@@ -0,0 +1,115 @@
+= Releasing INCEpTION
+
+== Prerequisites
+
+When releasing and publishing the resulting artifact, make sure that you have actually the rights
+to publish new artifacts to the target repository. See e.g.
+https://maven.apache.org/settings.html#Servers[the Maven documentationon authentication] or
```

Comment:
- typo: Maven documentation
- the link to the DKPro repositories setup confused me because it does not say anything about authentication to the repository